tests/keras_tests/feature_networks_tests/feature_networks/activation_bias_correction_test.py

```diff
@@ -27,171 +28,58 @@
-        # which adjusts the bias correction values to zero.
-        return QuantizationConfig(weights_bias_correction=False,
-                                  weights_second_moment_correction=False,
-                                  activation_bias_correction=True)
-
-    def create_networks(self):
-        inputs = layers.Input(shape=self.get_input_shapes()[0][1:])
-        x = layers.Activation('swish')(inputs)
-        x = layers.Flatten()(x)
-        x = layers.Reshape(target_shape=(8, 8, 3))(x)
-        outputs = layers.Conv2D(filters=3, kernel_size=2, use_bias=True, bias_initializer='ones')(x)
+        outputs = self.linear_layer(x)
         return keras.Model(inputs=inputs, outputs=outputs)
 
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
-        float_conv_layers = get_layers_from_model_by_type(float_model, layers.Conv2D)
-        quantized_conv_layers = get_layers_from_model_by_type(quantized_model, layers.Conv2D)
-
-        bias = float_conv_layers[-1].bias
-        bias_after_activation_bias_correction = quantized_conv_layers[-1].layer.bias
-
-        self.unit_test.assertTrue(np.array_equal(bias, bias_after_activation_bias_correction),
-                                  msg=f"Error in activation bias correction: expected no change in the bias value in "
-                                      f"case of conv2d with kernel 2.")
+        float_linear_layers = get_layers_from_model_by_type(float_model, type(self.linear_layer))
+        quantized_linear_layers = get_layers_from_model_by_type(quantized_model, type(self.linear_layer))
+
+        bias = float_linear_layers[-1].bias
+        bias_after_activation_bias_correction = quantized_linear_layers[-1].layer.bias
+
+        if getattr(float_linear_layers[-1], KERNEL_SIZE, None) in [None, 1, (1, 1)]:
+            if self.activation_bias_correction_threshold > 1e8:
+                self.unit_test.assertTrue(np.array_equal(bias, bias_after_activation_bias_correction),
+                                          msg=f"Error in activation bias correction: expected no change in the bias "
+                                              f"value in case of activation_bias_correction_threshold "
+                                              f"{self.activation_bias_correction_threshold}.")
+            else:
+                self.unit_test.assertFalse(np.array_equal(bias, bias_after_activation_bias_correction),
+                                           msg=f"Error in activation bias correction: expected a change in the bias "
+                                               f"value.")
+        else:
+            self.unit_test.assertTrue(np.array_equal(bias, bias_after_activation_bias_correction),
+                                      msg=f"Error in activation bias correction: expected no change in the bias value "
+                                          f"in case of conv with kernel 2.")
```

Comment:
I think you should add an output compare, so that we make sure that the quantized model is built correctly.